# HG changeset patch
# User John D
# Date 1463845227 14400
#      Sat May 21 11:40:27 2016 -0400
# Node ID ff2b9f7069c5e1bd1a1b5c68d2b561b884d9a7af
# Parent  31b4b614ed550f2038044ce8864d6c9519bf3edb
* liboctave/util/oct-glob.cc: remove stray '`'

diff --git a/liboctave/util/oct-glob.cc b/liboctave/util/oct-glob.cc
--- a/liboctave/util/oct-glob.cc
+++ b/liboctave/util/oct-glob.cc
@@ -109,17 +109,17 @@ namespace octave
                       retval.resize (k+n);
 
                       for (int j = 0; j < n; j++)
                         {
                           std::string tmp = matches[j];
 
 #if defined (OCTAVE_HAVE_WINDOWS_FILESYSTEM)    \
   && ! defined (OCTAVE_HAVE_POSIX_FILESYSTEM)
-`                          std::replace_if (tmp.begin (), tmp.end (),
+                          std::replace_if (tmp.begin (), tmp.end (),
                                            std::bind2nd (std::equal_to<char> (),
                                                          '/'),
                                            '\\');
 #endif
 
                           retval[k++] = tmp;
                         }
                     }
